docs/operations/ip-blocking.md

```diff
@@ -0,0 +1,62 @@
+# Blocking IP addresses
+
+This describes a mechanism to block access to EF-CMS, which can be used when an IP address is causing unexpected or unusual traffic. This is an augmentation to other, automated rate limiting and security measures which may also block access, and at best will be used sparingly.
```

Comment:
nit: should `DAWSON` be used instead of `EF-CMS`?

Reply:
Good question. I would say DAWSON is the currently deployed production environment of EF-CMS. There could be others if this repo was forked and repurposed, and so it's probably suitable to use EF-CMS as a generic reference. Curious what others think.